terraform/environments/analytical-platform-compute/lakeformation-permissions.tf

```diff
@@ -56,3 +56,8 @@ resource "aws_lakeformation_permissions" "cadet_domain_table_data" {
     }
   }
 }
+
+resource "aws_lakeformation_lf_tag" "sensitive" {
+  key    = "classification"
+  values = ["sensitive"]
+}
```

Comment:
```suggestion
```

Reply:
If you add it here: https://github.com/ministryofjustice/modernisation-platform-environments/blob/60f4ce733d04d2c397047667faba234233798d2c/terraform/environments/analytical-platform-compute/environment-configuration.tf#L137 It'll take care of the permissions too.